app/controllers/users_controller.rb

```diff
@@ -2,12 +2,20 @@ class UsersController < ApplicationController
   before_action :authenticate_user!
   
   def index
+
   end
 
   def update
   end
 
   def show
-    @user = User.find(params[:id])
+    @user = current_user
+    @list = @user.lists
```

Comment:
Since the user doesn't have lists, what should this be? How would we get a list of all the users items?